src/protocol/templates/ping_protocol.json

```diff
@@ -515,7 +515,7 @@
                         {
                             "name": "gain_index",
                             "type": "u32",
-                            "description": "The current gain setting. 0: 0.6dB, 1: 1.8dB, 2: 5.5dB, 3: 12.9dB, 4: 30.2dB, 5: 66.1dB, 6: 144dB"
+                            "description": "The current gain setting. 0: 0.6, 1: 1.8, 2: 5.5, 3: 12.9, 4: 30.2, 5: 66.1, 6: 144"
```

Comment:
Why are we removing the units here ?

Reply:
I will forward you the internal email. This is absolute gain.